src/components/CreateNote/CreateNote.jsx

```diff
@@ -0,0 +1,128 @@
+import "./create-note.css";
+import {
+  PushPinIcon,
+  PushPinOutlinedIcon,
+  AddIcon,
+  CloseIcon,
+} from "../../Icons/Icons";
+import { useState } from "react";
+import { v4 as uuid } from "uuid";
+import { useNoteContext } from "../../context/notes-context";
+
+const CreateNote = () => {
+  const { notes, setNotes, setShowNote } = useNoteContext();
+  const [notesData, setNotesData] = useState({
+    id: uuid(),
+    title: "",
+    note: "",
+    isPinned: false,
+    noteEdit: false,
+    color: "",
+    priority: "",
+    label: "",
+    isTrashed: false,
+    isArchived: false,
+    deletedFromTrash: false,
+    createdAt: new Date(),
+  });
+
+  console.log(notes);
+
+  return (
+    <div className="create-note" style={{ background: notesData.color }}>
+      <div className="note-head">
+        <input
+          className="note-input"
+          type="text"
+          placeholder="Title"
+          onChange={(e) =>
+            setNotesData((prev) => ({ ...prev, title: e.target.value }))
+          }
+        />
+        {notesData.isPinned ? (
+          <PushPinIcon
+            sx={{ color: "white" }}
+            onClick={() =>
+              setNotesData((prev) => ({ ...prev, isPinned: !prev.isPinned }))
+            }
+          />
+        ) : (
+          <PushPinOutlinedIcon
+            sx={{ color: "white" }}
+            onClick={() =>
+              setNotesData((prev) => ({ ...prev, isPinned: !prev.isPinned }))
+            }
+          />
+        )}
+      </div>
+      <div className="note-body">
+        <textarea
+          className="note-text-area"
+          placeholder="Take Note"
+          onChange={(e) =>
+            setNotesData((prev) => ({ ...prev, note: e.target.value }))
+          }
+        />
+      </div>
+      <div className="note-footer">
+        <select
+          name="label"
+          id="select-label"
+          onChange={(e) =>
+            setNotesData((prev) => ({ ...prev, label: e.target.value }))
+          }
+        >
+          <option value="">choose label</option>
```

Comment:
the value should not be empty I think. If not needed you can remove or you have forgotten to place value into it